src/scenes/Languages/Detail/LanguageDetail.tsx

```diff
@@ -52,6 +52,11 @@ const useStyles = makeStyles(({ spacing }) => ({
   hidden: {
     visibility: 'hidden',
   },
+  projectSection: {
+    '& > *': {
+      marginBottom: spacing(2),
+    },
```

Comment:
the list items already have this applied. Maybe we can just use paragraph or gutterBottom on the title instead of ths?

Reply:
yeah good point.  The gutterBottom isn't big enough so pointed to `h3`

Reply:
Did you try paragraph? If so, then let's just move the css class to typography instead unnecessarily going up to parent section and then down in css.

Reply:
Didn't know about `paragraph` but that actually works perfect
<img width="411" alt="Screen Shot 2020-11-05 at 8 49 33 AM" src="https://user-images.githubusercontent.com/43487134/98270976-4ae9b700-1f44-11eb-8002-80ad41251ec5.png">
<img width="746" alt="Screen Shot 2020-11-05 at 8 49 39 AM" src="https://user-images.githubusercontent.com/43487134/98270979-4c1ae400-1f44-11eb-923e-19ccf41afc58.png">